src/python/pants/reporting/BUILD

```diff
@@ -8,6 +8,8 @@ python_library(
     '3rdparty/python:ansicolors',
     '3rdparty/python:pystache',
     '3rdparty/python:six',
```

Comment:
One more change. Because I asked you to use `future.utils` instead of `six`, this needs to be changed to `'3rdparty/python:future',`. Sorry I forgot to mention you needed to do this!

For context, both `six` and `future.utils` would have worked for the use case; you can think of `future` as a souped up version of `six`, both trying to provide Py2 vs Py3 compatibility. But I requested you to change to `future.utils` for consistency. `six` has some problematic things about it, such as how it backports `StringIO`. So in general, we try to ban `six` as much as possible in our codebase and exclusively use the `future` library.

Reply:
For `src/python/pants/reporting/BUILD` I didn't add `six` to build file, it was there before I did any changes. Also, this build file has `'3rdparty/python:future'` dependency.
I added and then deleted PY2 import in `tests/python/pants_test/reporting/test_reporting_integration.py` file. But in it's build file `tests/python/pants_test/reporting/BUILD` there is no `six` and there is `'3rdparty/python:future'`

Reply:
Ah sounds good! I didn’t look as closely as I should have - sorry about the confusion and congrats on your first PR :)